compiler/ast.nim

```diff
@@ -1068,7 +1158,7 @@ proc isCallExpr*(n: PNode): bool =
 
 proc discardSons*(father: PNode)
 
-type Indexable = PNode | PType
+include astmut
```

Comment:
can you make this an import? include have nasty gotchas

Reply:
This is an include because I need to export parts of it.

Reply:
no need for include, it's a standard pattern:
```nim
import astmut
export astmut
```